python/sparkdl/transformers/keras_applications.py

```diff
@@ -16,30 +16,33 @@
 from abc import ABCMeta, abstractmethod
 
 import keras.backend as K
-from keras.applications import inception_v3
+from keras.applications import inception_v3, xception
 from keras.applications.imagenet_utils import decode_predictions
 import tensorflow as tf
 
 from sparkdl.transformers.utils import (imageInputPlaceholder, InceptionV3Constants)
 
 
-KERAS_APPLICATION_MODELS = set(["InceptionV3"])
-
+KERAS_APPLICATION_MODELS = set(["InceptionV3", "Xception"])
 
 """
 Essentially a factory function for getting the correct KerasApplicationModel class
 for the network name.
 """
 def getKerasApplicationModel(name):
     if name not in KERAS_APPLICATION_MODELS:
-        raise ValueError("%s is not a supported model. Supported models: %s" % name,
-                         str(KERAS_APPLICATION_MODELS))
+        raise ValueError("%s is not a supported model. Supported models: %s" %
+                         (name, str(KERAS_APPLICATION_MODELS)))
 
     if name == "InceptionV3":
         return InceptionV3Model()
+    elif name == "ResNet50":
+        return ResNet50Model()
+    elif name == "Xception":
+        return XceptionModel()
```

Comment:
We can use a dictionary for `KERAS_APPLICATION_MODELS`, something like:
```
KERAS_APPLICATION_MODELS = {"ResNet50": ResNet50Model,
...
}
```

that way this block becomes:

```
try:
    return KERAS_APPLICATION_MODELS[name]()
except KeyError:
   raise ValueError(...)
```
.